code/src/main/java/com/onlinecontacttracing/servlets/get-data/GetPositiveUserInfoServlet.java

```diff
@@ -0,0 +1,61 @@
+package com.onlinecontacttracing.authentication;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdToken.Payload;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import com.onlinecontacttracing.storage.PotentialContact;
+import static com.googlecode.objectify.ObjectifyService.ofy;
+import com.onlinecontacttracing.storage.PositiveUser;
+import java.util.Optional;
+import java.util.ArrayList;
+
+@WebServlet("/get-positve-user-info")
+public class GetPositiveUserInfoServlet extends CheckForApiAuthorizationServlet {
+
+  String getServletURIName() {
+    return "/get-positve-user-info";
+  }
+  
+  
+  /*
+  * This servlet will retrieve data from the People Api
+  * Additionaly it will forward the request to the servlet for Calendar Api
+  * Once both are done, the servlet will merge contact data sets
+  */
+  void useCredential(Credential credential, HttpServletResponse response) throws IOException, InterruptedException {
+    // Execute runnable to get people data
+    ArrayList<PotentialContact> contactsFromPeople = new ArrayList<PotentialContact>();
+    Thread peopleInfo = new Thread(new GetPeopleDataForPositiveUser(credential, contactsFromPeople));
+    Thread contactInfo = new Thread(new GetCalendarDataForPositiveUser(credential));
+
+    peopleInfo.start();
+    contactInfo.start();
+
+    
+    peopleInfo.join();
+    contactInfo.join();
+
+    // Load PositiveUserContacts from objectify
+    // call mergeContactListsFromPeopleAPI(contactsFromPeople)
```

Comment:
Incomplete code again.

Reply:
this is meant for when all the data is acquired after implementing the runnables (seperate PRs)